config/jobs/update-github-teams/peribolos-syncer-charts.yaml

```diff
@@ -0,0 +1,61 @@
+# Place here other repository OWNERS to synchronize GitHub Team Peribolos configuration from.
+#
+# More on this here: https://github.com/falcosecurity/peribolos-syncer/blob/v0.1.0/docs/peribolos-syncer_sync_github.md#peribolos-syncer-sync-github
+#
+postsubmits:
+  falcosecurity/charts:
+  - name: peribolos-syncer-post-submit
+    branches:
+    - ^main$
+    decorate: true
+    max_concurrency: 1
+    skip_report: false
+    run_if_changed: 'OWNERS$'
+    spec:
+      containers:
+      - image: gcr.io/maxgio92/peribolos-syncer:0.2.0
```

Comment:
Before using the `falcosecurity` package, we need https://github.com/falcosecurity/peribolos-syncer/pull/11.